### PR TITLE
Add support to edit the Simplivity policy rule

### PR DESCRIPTION
**Pull request by svt-hchaudhari**
Signed-off-by: Hemant Chaudhari <hchaudhari@hpe.com>

### Description
Add support to edit the Simplivity policy rule

### Issues Resolved
None

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass(`$ tox`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: simplivity/resources/policies.py [hunk 1]
```diff
@@ -199,3 +199,42 @@ def get_rule(self, rule_id):
         """
         resource_uri = "{}/{}/rules/{}".format(URL, self.data["id"], rule_id)
         return self._client.do_get(resource_uri)
+
+    def edit_rule(self, rule_id, rule, timeout=-1):
+        """Edits an existing policy rule
+        Args:
+            rule_id: Rule id to be edited
+            rule: Any or all of the backup policy rule fields can change, except destination_name, id, number,
```
Comment (sijeesh) [inline simplivity/resources/policies.py]: mention the expected type of this argument

Comment (svt-hchaudhari) [reply simplivity/resources/policies.py]: Updated as suggested

**Review by sijeesh — APPROVED** [review]
LGTM

**Comment by sijeesh** [comment]
@svt-hchaudhari, please resolve the conflicts.

**Pull request merged**